### ISSUE TITLE
Copyright string must be ascii

### ISSUE THREAD

**Issue by davelab6** [mozilla/Fira#149]
OTF fonts can't have non-ascii characters in their CFF table metadata, and while the LICENSE file has 

    Digitized data copyright (c) 2012-2015, The Mozilla Foundation and Telefonica S.A.

the font files have

    Digitized data copyright © 2012-2015, The Mozilla Foundation and Telefonica S.A.

That `©` is bad, and the string used in the LICENSE file should be used.

**Comment by Pomax** [comment]
although unfortunately, (c) (with parentheses rather than as single symbol) is not a legal equivalent to the copyright symbol in the US and the UK at the very least, so the other question is "is this still worth putting in the CFF block at all", as OpenType string metadata comes from the `name` table, not from the CFF block.

**Comment by davelab6** [comment]
Copyright notices are not required, so I'm unconvinced that the copyright symbol and an ascii facsimile of it are meaningfully different

**Comment by Pomax** [comment]
quite, although the more important part is "this probably shouldn't be in the CFF block at all". The CFF spec is deliciously vague about which string format is expected (or allowed), but OpenType at least wants all strings that have meaning outside the font in the name table. Strings in the CFF block should never make it out of the font in the first place, so any license text in it is supposed to never even been seen, thus mostly serving to increase the filesize.

**Comment by davelab6** [comment]
Agree. I think the OT spec is up for revision soon; will you make a
proposal? :)

**Comment by Pomax** [comment]
I could, although the question is also a bit *where* that proposal should go. Ideally it'd be an update to the CFF spec, not the OpenType spec, to add a section on "when used inside an OpenType font, the following relaxations/rules apply: ...", as the OpenType spec itself is already pretty explicit when it comes to where strings should go.

Putting that aside (as a thing to ask on the OpenType list =), when used in an OpenType font the CFF TOP DICT actually only needs to contain the offsets to the rest of the CFF data structures. None of the "metadata" values are in fact needed for the OpenType font to work, so strings for version, font family, copyright, etc, can all be safely removed.

**Comment by adrientetar** [comment]
(c) has no legal value, but if you write Copyright you don't need the symbol anyway.

**Comment by davelab6** [comment]
That's also my understanding.

**Comment by patrykdesign** [comment]
@carrois
